doc/examples/141-https-self-signed/config.json

```diff
@@ -2,7 +2,7 @@
         "api": {
             "baseUrl": "https://mock-server-https-self-signed-proxy:443/141-https-self-signed/",
-            "caCertificate": "-----BEGIN CERTIFICATE-----\nMIIDNDCCAhwCCQCSteKUCHafQTANBgkqhkiG9w0BAQsFADBcMQswCQYDVQQGEwJV\nUzEPMA0GA1UECAwGRGVuaWFsMRQwEgYDVQQHDAtTcHJpbmdmaWVsZDEQMA4GA1UE\nCgwHRGlzUm9vdDEUMBIGA1UEAwwLQ29tcGFueVJvb3QwHhcNMjAwOTA3MTMzNDAy\nWhcNMjMwNjI4MTMzNDAyWjBcMQswCQYDVQQGEwJVUzEPMA0GA1UECAwGRGVuaWFs\nMRQwEgYDVQQHDAtTcHJpbmdmaWVsZDEQMA4GA1UECgwHRGlzUm9vdDEUMBIGA1UE\nAwwLQ29tcGFueVJvb3QwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDQ\n5NSB5495dFt4X3TGrIV5iyqe//bHFOUcGM9eCL385csgskPFsX6HTHOMy4EhaNnD\nADy9xEJ9MMeeoN8y+K63LbHKvkaqH0O7AAF0WLtiBPozkta8InPa3qHyZKSHDlQT\niNKolFmWrAiGC25Wz95KNrX3XMwQKAHfclnsK0tKF4N0H4e4Eox81KNzUuNmfcJ0\n/WAF2XBldiaj8ggIJAK56pdHk3+RdV0MhLqVeCjzv216KyL/2upjAu0wbqLdfrHR\n9GjNBT/dIYGXXRZI6P4pivyTpTrovIGpO1gc0OVrfk569P5jo0ZH0M0l52m0QSXD\nvJteV6lf6OF2Dge6+ImdAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAAnHmwrzodgS\n9rmjIZTuQdvKJcNGD7GZbxKtSOLwSe0G4/BDC+42TarYV6CBliOdMbLTflQmpYRJ\nfrQ94vtf4QoF0SvZnFs5EgO6152uii4W/1cgHbgX1zXOrQht9K64Nc4r7T0YEriE\n0mg14G9auB7kTrvSGRbMZ81QQ55tJCx0/7pg/9Xn+cLmjUpAnBlMgNgx/jMjYF1O\nFP01Z3rtCcO9hvVpWkp5duhmDWkS5fEoC2hklXRheH298SWPa/ld2ZT/Lyr5dhJ5\n7we+/PtzXBkwPdkrsVd4LNTVnmQTuvXIjDKxa0L4ZGHWNAI02yYwz5OfT64nsbAI\nFhUCy6kGY60=\n-----END CERTIFICATE-----\n"
+            "caCertificate": "-----BEGIN CERTIFICATE-----\nMIIEZzCCAk8CFFgTlrKabPLuSTRvTrym3/0Nl5O9MA0GCSqGSIb3DQEBCwUAMHAx\nCzAJBgNVBAYTAlhYMRUwEwYDVQQHDAxEZWZhdWx0IENpdHkxHDAaBgNVBAoME0Rl\nZmF1bHQgQ29tcGFueSBMdGQxLDAqBgNVBAMMI21vY2stc2VydmVyLWh0dHBzLXNl\nbGYtc2lnbmVkLXByb3h5MB4XDTIwMTEzMDE5NDA1NFoXDTQ4MDQxNzE5NDA1NFow\ncDELMAkGA1UEBhMCWFgxFTATBgNVBAcMDERlZmF1bHQgQ2l0eTEcMBoGA1UECgwT\nRGVmYXVsdCBDb21wYW55IEx0ZDEsMCoGA1UEAwwjbW9jay1zZXJ2ZXItaHR0cHMt\nc2VsZi1zaWduZWQtcHJveHkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIB\nAQDDf+vS3gnOU8dibGh1VTrEPry172oWPEp4Ybn31h5+NWpnmKWufGRc3HeZ0/cJ\nJ4V4HQmiCXc6yGxStx3EMaGT11DzSvbguZChSj+DxckFgxuox4wL7bOzScM/yWkL\nBK21wxLl92RiKlgBvdt3bDyJbOraxGSKBNRBBLM6Wh1JWc06YxF44e0T3B7H/Qtn\nWPXmsi1YPbYAxlkeoeJ6WNwfzEuLMnFf/fWgL2TgFwLIfhqOzlyU2CyHSAhOX3Ny\n5x3wIMxdT51MW601iCKhdluN6aNM0lTgHNAiz7RD0ZFtqWDN0XvOzTU+7jxA0926\nqtk18g9XU9R3WDNw1p6FNbdrAgMBAAEwDQYJKoZIhvcNAQELBQADggIBAFWQJM3l\nWimy7DYKn3Dy1RH9Pd4mIT0TJ4V1T/eszbWB7YRDdHuYyf7Sk0QDZPLMqR2sYxMc\nShdrQpOgU/o/iIkhwAjW6LzbuiQJULd74u3J08RPAEpSSJFJwQD4ZU2B7nhoqxft\noxUzoytfUSnsMA2EUJFYsPvkygyCihudNR+44ONq771W9HfvFNH3YdQyxjI/RtwT\nCiYdxQFiS5UvWCBbJAVmRWLm1gnJbG+MYBye4mGfyiZ3iMq/Wnb++favC8jNDYSO\nokha+LQBpscQZ3fbn5srh7uQGXOkQjT8gsd31neEFfXbVlrnUsHAXMHHgUgNrF+k\nPVp0321fzmvwxGwjB4/jHd+1ImDwcDJ6LseKwtbriN5EDbwEV/mIME2/MjFfqFhV\nM3AkhgW/sY4KvAD+FZ77U3xR4iIQgknjAPEYgqKDHF0h0295fwhRl7wmw/nO6Z8e\nIHP+YOH7gcdYskr2cnXXD9J2z0T3/VZUyZjnrmVqCdWq4hggea1qKYw6qHbtuETV\n9CCxLE+LTrFfYHHkQI2Jg8E+NDApjOodFoSaLphTzK8FZoyn9ttrR8UcFFMW95nt\n9POau3WtvfYvAve0h4UYj9ZMotNnTFWyhmgn9gu6gvuyVfvUKJlsW+KQbJ8bJraN\n0SLVT2nUF/WUpJde3jvWs+OtQCQEUZds5qfy\n-----END CERTIFICATE-----\n"

```

Comment:
To se meni kvuli tomu updatu PHP? Neprestanout pak fungovat nejake certifikaty?

Reply:
@odinuv nie, len ked som tam minule pridaval ten certifikat - novy test ... tak som mu musel nastavit malu platnost, ... a teraz mi padal test, ze certifikat uz neplati (aj ked je self-signed, ale musi platit) .... tak som tie certifikaty iba pregeneroval na `10000 days`, nesuvisi to s nicim inym.